app.go

```diff
@@ -609,12 +609,17 @@ func (app *App) SetTLSHandler(tlsHandler *TLSHandler) {
 // Name Assign name to specific route.
 func (app *App) Name(name string) Router {
 	app.mutex.Lock()
+	for _, routes := range app.stack {
+		for _, route := range routes {
+			if route.Path == app.latestRoute.path {
+				route.Name = name
+			}
+		}
+	}
 
 	latestGroup := app.latestRoute.group
```

Comment:
Can you add test cases for group and use

Reply:
I added a test for Group locally and it failed. Sadly, I'm not able to get the group names to work as group was always <nil>

Sorry about this